addons/crm/models/res_partner.py

```diff
@@ -12,6 +12,7 @@ class Partner(models.Model):
     opportunity_ids = fields.One2many('crm.lead', 'partner_id', string='Opportunities', domain=[('type', '=', 'opportunity')])
     meeting_ids = fields.Many2many('calendar.event', 'calendar_event_res_partner_rel', 'res_partner_id', 'calendar_event_id', string='Meetings', copy=False)
     opportunity_count = fields.Integer("Opportunity", compute='_compute_opportunity_count')
+    partner_opportunity_ids = fields.Many2many('crm.lead', compute="_compute_opportunity_count", string='Partner Opportunities')
```

Comment:
I would rather name it:
```suggestion
    opportunity_count_ids = fields.Many2many('crm.lead', compute="_compute_opportunity_count", string='Technical field used for stat button')
```

So it's clear it is only intended to be used for the stat button.